### DISCUSSION TITLE
page lagging

### DISCUSSION THREAD

**Discussion by JerryWn12** [waylaidwanderer/node-chatgpt-api#61] [Discussions]
when browser playing cli.svg in readme.md, the page becomes lagging, happens on android and windows, Chrome browser, anyone faced that?

**Comment by waylaidwanderer** [comment]
I haven't had issues with Firefox myself, but maybe I should turn it into a gif instead... Someone else said it was laggy for them too.

**Reply by TomBener** [reply]
Same issue on Safari. Please fix it.

**Reply by waylaidwanderer** [reply]
Fixed with https://github.com/waylaidwanderer/node-chatgpt-api/commit/e9c5771c561476d65d27b3aace8870ea5e4379dd (cc @JerryWn12)
